webapp/src/reducers/websocketEvent/index.ts

```diff
@@ -0,0 +1,19 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
```

Comment:
do we need this?
the template has too many things we should keep only those for now which are needed, code that are not needed now should be removed and can be added when required otherwise it will increase the bundle size.